### ISSUE TITLE
Export Duration constructor

### ISSUE THREAD

**Issue by shayded-exe** [moment/moment#5605]
**Is your feature request related to a problem? Please describe.**
I want to use the Duration type with my graphql library, which requires passing the type to a decorator.

```ts
import { Duration } from 'moment';

export class SomeType {
  @Field(() => Duration)
  //           ^^^^^^^^ ts(2693)
  term!: Duration;
}
```

This gives the error: `'Duration' only refers to a type, but is being used as a value here.ts(2693)`.

This of course is because moment only exports `Duration` as an interface in the typings.

**Describe the solution you'd like**
The `moment` module should export the Duration constructor defined in `src/lib/duration/constructor.js`.

`moment.d.ts` should declare `Duration` as a class instead of an interface.

**Describe alternatives you've considered**
You could do something like this:
```ts
import moment from 'moment';

const DurationConstructor = moment.duration().constructor;
```
but it isn't very nice and you'd still need to do some additional work for the typings.

**Comment by zahra-abdullah** [comment]
@marwahaha @PachowStudios Ill be working on it!

**Comment by JuanVqz** [comment]
HI! are you saying that
should I add `default` [here](https://github.com/moment/moment/blob/develop/src/lib/duration/constructor.js#L5)?
I mean 
```js
export default function Duration(duration) {...}
```
if not can you help me with an example in order to solve it? can I grab it?

**Comment by rshea0** [comment]
@JuanVqz No, instead you need to add the Duration constructor to the top level moment export.

https://github.com/moment/moment/blob/develop/src/moment.js
```js
moment.Duration = Duration;
```

**Comment by mayralgr** [comment]
@rshea0 is this still open? can you clarify this a little bit more for me, please. I would like to give it a try

**Comment by Mahdhir** [comment]
Trying to work on this.

Tried to build the project as it is before making any changes, Running into this error during `grunt`

```
Running "exec:checkMonthsParseIssue" (exec) task
>> $(ls was unexpected at this time.
>> Exited with code: 1.
>> Error executing child process: Error: Process exited with code 1.
Warning: Task "exec:checkMonthsParseIssue" failed. Use --force to continue.

Aborted due to warnings.
```

Can someone guide me on this? 
OS: Windows
Terminal: bash

**Comment by nadasuhailAyesh12** [comment]
can I work on this issue ,please. I need a merged PR Contribution for my university assignment

**Comment by mattjohnsonpint** [comment]
Hi. I don't want to potentially introduce breaking TypeScript changes in Moment 2.x, and I'll be taking a different approach with TypeScript in Moment 3.0 fairly soon, so closing this. Thanks for your patience!
